apps/network-alerts/js/notification/notification.js

```diff
@@ -18,12 +18,17 @@ function onNotification(message) {
     return;
   }
 
-  var title = Utils.parseParams(message.imageURL).titleID;
+  var title = null;
+  if (message.data) {
+    title = message.data.title;
+  }
+  else {
+    title = Utils.parseParams(message.imageURL).titleID;
```

Comment:
Because when I don't keep the backward compatibility sometime notification-test.js fails because message.data.title doesn't exist. With this it was working well ^^